app/src/renderer/connectors/lcdClientMock.js

```diff
@@ -353,13 +355,13 @@ let state = {
       description: `custom text proposal description`,
       initial_deposit: [
         {
-          denom: `stake`,
+          denom: `steak`,
```

Comment:
didn't we want to move towards `stake`?

Reply:
yeah but it's not going to be used until GoS and the denom for governance is still steak

Reply:
sure, but this is just the mock implementation and not the real network. does it conflict with settings for the real network?

Reply:
not really, but I'd have to add another proposal or refactor it to change that, bc it'll probably break other unit tests that use proposals. Can I do that on the scope of #1378 ?

Reply:
Sure. I just did that PR. Let's see afterwards what breaks. 💥